lib/elixir/lib/enum.ex

```diff
@@ -1180,8 +1184,14 @@ defmodule Enum do
 
   """
   @spec into(Enumerable.t(), Collectable.t()) :: Collectable.t()
+  def into(enumerable, collectable)
+
+  def into(enumerable, []) do
+    to_list(enumerable)
+  end
+
   def into(enumerable, collectable) when is_list(collectable) do
```

Comment:
We can get rid of this clause altogether, no?

Reply:
Good catch, removed!